assets/styles/components.scss

```diff
@@ -512,6 +512,25 @@ tr.button-transparent {
   --text-color: var(--color-brand-inverted);
 }
 
+.stateful-button {
+  display: grid;
+  align-items: center;
+  justify-items: center;
+
+  & > span {
+    grid-column-start: 1;
+    grid-row-start: 1;
+    visibility: hidden;
+    display: flex;
+    align-items: center;
+    justify-items: center;
+
+    &.active {
+      visibility: visible;
+    }
+  }
+}
+
```

Comment:
what is this used for?

Reply:
In some languages buttons will change length depending on their state, this was made in previous iteration of this PR as a test and was accidentally ported over (this part only, not the changes to layout). It allows to provide all the states for the button at the same time and let CSS automatically calculate its width, so when state changes (e.g. follow ⇒ followed) there won't be any layout shifts, which is generally not a problem for English, but might be problem for say, German. I believe you can see this problem on Twitter when you hover or click follow button. I'm not yet sure this is required, this was just a test and I believe only used once in previous iteration for copy token button in settings.